tests/generate/test_generate_generators.test_generate_generators_no_toml.approved.txt

```diff
@@ -0,0 +1,9 @@
+Usage: algokit generate [OPTIONS] COMMAND [ARGS]...
```

Comment:
Should there be some difference when there is no toml?

Reply:
@robdmoore  see all tests under test_generate_custom_generate_commands.* prefix